docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/amazon-quicksight.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Amazon QuickSight
+description: Amazon QuickSight overview
+template: concept-topic-template
+last_updated: Oct 24, 2024
+---
+
+Amazon QuickSight brings customizable and powerful business intelligence tools to the Back Office.
+
+## Data sources and data sets
+
+This feature supports data from Spryker projects and third-party systems. When you set up Amazon QuickSight, the data from your project is available by default, and you can add other data sources if needed.
+
+![QuickSight-datasets](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/amazon-quicksight.md/qs-data-sets.png)
+
+For more details on data sources, see [Supported data sources](https://docs.aws.amazon.com/quicksight/latest/user/supported-data-sources.html).
+
+### Extending project data for analytics
+
+By looking at datasets, you can see what data from your project is available for adding to analytics. By default, for each dataset, only the data that can be useful for analytics is available from a respective database table. For example, the Customer dataset doesn't include customer passwords.
```

Comment:
```suggestion
By looking at datasets, you can see what data from your project is available for analytics. By default, only the data that can be useful and secure for analytics is available from the respective database tables for each dataset. For example, the Customer dataset doesn't include customer passwords or names.
```

Reply:
My bad, I wanted to say that customer names are available. First and last names are in bold

Reply:
ah ok, thank you for pointing this. I sent you the file in Slack which is more accurate.